src/richie/apps/search/__init__.py

```diff
@@ -10,7 +10,8 @@
 default_app_config = "richie.apps.search.apps.SearchConfig"
 
 ES_CLIENT = ElasticsearchClientCompat7to6(
-    [getattr(settings, "RICHIE_ES_HOST", "elasticsearch")]
+    [getattr(settings, "RICHIE_ES_HOST", "elasticsearch")],
+    **getattr(settings, "RICHIE_ES_KWARGS", {}),
```

Comment:
should we use names that are coherent with the related variable names? ie `RICHIE_ES_CLIENT_KWARGS` and `RICHIE_ES_INDICES_CLIENT_KWARGS`. Maybe these variables should be renamed if their name is not good?

Reply:
Yes, I've renamed them. Also included a new setting for each elastic search lib call so it was possible to tune each call.